### DISCUSSION TITLE
[Feature] Use arrow keys on app

### DISCUSSION THREAD

**Discussion by mmomjian** [immich-app/immich#7665] [Feature Request]
### The feature

On iOS devices, the arrow keys (from an attached keyboard on an iPad) do not scroll between images in the Immich app. They do work on the website.

Solution: monitor for keyboard presses to scroll images in the iOS app.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by benosman** [comment]
The arrow keys also do not navigate in the android app.

Additionally, with a connected mouse you can't scroll between images by by dragging the image off the screen. It does let you drag it, but it returns to the center of the viewport. 

Google photos app handles both keyboard and mouse navigation.

**Comment by Klar** [comment]
I would very much like this feature and would like to add to have the "F" key added as well for the Star like feature.
